content/pages/disability-benefits/claims-process.md

```diff
@@ -18,6 +17,12 @@ relatedlinks:
   - url: /disability-benefits/claims-process/date/
     title: How does VA determine the effective date of a disability?
     description: ""
+  - url: /disability-benefits/claims-process/what-happens-after-rating/
+    title: You have a disability rating. What&rsquo;s next?
+    description: ""  
+  - url: /disability-benefits/claims-process/claim-exam/
```

Comment:
@crwallace https://github.com/department-of-veterans-affairs/vets-website/pull/2992